source/Octopus.Client/Repositories/Async/UserRepository.cs

```diff
@@ -21,7 +21,7 @@ public interface IUserRepository :
         Task SignOut();
         Task<UserResource> GetCurrent();
         Task<SpaceResource[]> GetSpaces(UserResource user);
-        Task<ApiKeyCreatedResource> CreateApiKey(UserResource user, string purpose = null);
+        Task<ApiKeyCreatedResource> CreateApiKey(UserResource user, string purpose = null, DateTimeOffset? expires = null);
```

Comment:
Does null here mean it never expires? If so, it might be worth an Xml doc to point that out explicitly